onnxruntime/test/testdata/transform/fusion/reshape_fusion_gen.py

```diff
@@ -187,3 +187,35 @@ def save_model(graph, file_name):
 
 save_model(graph, 'reshape_fusion_with_graph_inputs.onnx')
 
+graph = helper.make_graph(
```

Comment:
Please add a case that some nodes of subgraph has more than one output: expect that the remove nodes will not remove it.